Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM srcd/lookout-sdk-ml:0.9.0
+FROM srcd/lookout-sdk-ml:0.11.0
```

Comment:
Oh jeez, this is going to put us into so much trouble. It contains the updated Modelforge which is quite incompatible with all the current models. There is nothing we can do, unfortunately.

Reply:
Yes I saw this this morning, and started to fix some stuff about it